llm/rag/build_rag.yaml

```diff
@@ -0,0 +1,27 @@
+name: build-legal-rag
+
+workdir: .
+
+resources:
+  memory: 32+  # Need more memory for merging embeddings
+  cloud: aws
+
+file_mounts:
+  /embeddings:
+    name: sky-rag-embeddings
```

Comment:
this may not work directly for a new user as the bucket can be occupied. How about we add a env for bucket name so the user can pass them in?